src/Pages/LandingPage/LandingPage.css

```diff
@@ -0,0 +1,139 @@
+.landing {
+  grid-area: contentarea;
+  display: flex;
+}
+
+.landing-left {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+  width: 100%;
+}
+
+.landing-header {
+  font-size: 3rem;
+  font-weight: bolder;
+  margin-top: 2rem;
+  color: var(--primary-color);
+}
+
+.landing-info {
+  font-size: 2rem;
+  width: 50rem;
+  text-align: center;
+  margin-top: 2rem;
+  color: var(--content-color);
+}
+
+.landing-btn {
+  border: none;
+  color: var(--white);
+  background-color: #3da4df;
+  padding: 1rem;
+  border-radius: 1rem;
+  margin-top: 2rem;
+  cursor: pointer;
+  font-size: 2rem;
+}
+
+.landing-btn:hover {
+  background-color: var(--primary-color);
+  transition: 0.2s ease-in-out;
+}
+
+.landing-link {
+  margin-top: 2rem;
+  color: var(--content-color);
+  cursor: pointer;
+  font-size: 1.5rem;
+}
+
+.landing-right {
+  height: 100%;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
```

Comment:
you could combine declarations for left and right, as they are almost same, save to reduce number of lines

Reply:
didint think about that I will try to implement it